variod.c

```diff
@@ -312,52 +311,9 @@ int main(int argc, char *argv[])
 
 					xcsoar_sock = create_xcsoar_connection();
 					break;
-
-				} else {
-					fprintf(stderr, "send failed\n");
 				}
 			}
 
-			// use specific data from received messge locally
-			// strtok will gobble up the content of client_message
-			next_sentence = strtok(client_message,sentence_delimiter);
-
```

Comment:
I believe this diff to `variod.c` does not belong to this PR. Please get rid of it (unless I'm mistaken).

Reply:
This was from the first method to make it handle multiple NMEA sentences from one recv().
In the meantime I came up with a more elegant, efficient and robust way to achieve the same goal.